pkg/controller/operators/olm/operator.go

```diff
@@ -1093,6 +1095,9 @@ func (a *Operator) handleClusterServiceVersionDeletion(obj interface{}) {
 		"phase":     clusterServiceVersion.Status.Phase,
 	})
 
+	logger.Debug("start deleting CSV")
```

Comment:
Do we need these?

Reply:
They are useful in tracking what's happening; this one is debug, so it doesn't clutter the log normally.

Reply:
Are you asking me to get rid of the extra logs?